website/docs/docs/cloud/use-visual-editor.md

```diff
@@ -30,14 +30,10 @@ To join the private beta, [register your interest](https://docs.google.com/forms
 Before accessing the editor, you should have a dbt Cloud project already set up. This includes a Git repository, data platform connection, environments, and developer credentials. If you don't have this set up, please contact your dbt Cloud Admin.
 
 To access the visual editor:
-- Type in the following URL, replacing the ACCOUNT_ID and ENVIRONMENT_ID with your own account and environment ID: `https://ACCESS_URL/visual-editor/ACCOUNT_ID/env/ENVIRONMENT_ID/`
-  - The environment ID must have had runs that generated catalogs in it.
 
-- For example, if my region is North America multi-tenant, account ID is 10, environment ID with a generated catalog run is 100, my URL should be:
+- Navigate to the left hand side panel and click **Develop**. From the menu, select **Visual Editor**.
 
-  - `https://cloud.getdbt.com/visual-editor/10/env/100/`
-
-<Lightbox src="/img/docs/dbt-cloud/visual-editor/visual-editor.png" width="80%" title="Visual editor interface that contains a node toolbar and canvas." />
+<Lightbox src="/img/docs/dbt-cloud/visual-editor/visual-editor.png" width="80%" title="Visual editor." />
```

Comment:
i see 'Visual Editor' in the img description, however sugg a more descriptive title for accessibility. so maybe: 

```suggestion
<Lightbox src="/img/docs/dbt-cloud/visual-editor/visual-editor.png" width="80%" title="Access Visual Editor by selecting 'Develop' from the navigation menu." />
```

Reply:
here's what i see

<img width="288" alt="Screenshot 2025-02-03 at 12 02 32" src="https://github.com/user-attachments/assets/b67faddc-f336-4b50-a1e5-92d20f8fc99d" />